api/src/certification/session-management/domain/usecases/create-certification-challenge-live-alert.js

```diff
@@ -19,6 +20,14 @@ const createCertificationChallengeLiveAlert = async function ({
 
   const answers = await answerRepository.findByAssessment(assessmentId);
 
+  const alreadyAnsweredChallenge = answers.find(
+    ({ challengeId: currentChallengeId }) => currentChallengeId === challengeId,
+  );
```

Comment:
Suggestion: Renommer en `isCurrentChallengeAlreadyAnswered` pour être plus explicite ? 
Ca impliquerait la modification suivante pour retourner un booléen.

```suggestion
  const isCurrentChallengeAlreadyAnswered = Boolean(answers.find(
    ({ challengeId: currentChallengeId }) => currentChallengeId === challengeId,
  ));
```